### DISCUSSION TITLE
running CUDA on mac or any non-nvidia gpu

### DISCUSSION THREAD

**Discussion by vinayakarsh** [karpathy/llm.c#220] [Q&A]
im new to CUDA.. how to run CUDA programs on mac or any non-nvidia gpu(s)/systems..

**Comment by wbmlr** [comment]
CUDA runs only on NVIDIA GPUs. That's why people use only NVIDIA GPUs and not other GPUs.

https://stackoverflow.com/questions/55320552/can-i-use-cuda-with-a-non-nvidia-gpu
